cscs-checks/microbenchmarks/gpu/memory_bandwidth/memory_bandwidth.py

```diff
@@ -5,23 +5,28 @@
 
 import reframe as rfm
 
-import library.microbenchmarks.gpu.memory_bandwidth as mb
 import cscslib.microbenchmarks.gpu.hooks as hooks
+from library.microbenchmarks.gpu.memory_bandwidth import *
 
 
-class BaseCSCS(rfm.RegressionMixin):
+class Base_CSCS(rfm.RegressionMixin):
```

Comment:
This name is too generic. I would make that simply private and name it anything. Also before changing the names, let's all settle on a style. I'm either all lowercase with underscores of camel. For sure, the actual tests should be lowercase with underscores, because they are more readable when showing up in listings and are easier to type. For internal classes we can keep the Python convention. For library test bases? I've no strong opinion.

Reply:
I agree with the internal CSCS convention to name the tests all lowercase and with underscores. I would also add to this convention that we should append `_check` at the end. This would be for the classes that are decorated with the `@rfm.simple_test` decorator.

For the libraries and other non-decorated classes, I would also vote for the underscore style, but here allowing upper cases when appropriate and mandatory at the beginning. It feels to me that we're breaking the law by naming a class with all lowercase. Other library users might have a strong opinion there.

Reply:
For example, in the gpu burn test, we would import the `GPU_burn` class from the library, but the decorated test would be 
```python
from library.microbenchmarks.gpu.gpu_burn import GPU_burn

@rfm.simple_test
class gpu_burn_check(GPU_burn):
   ...
```